### ISSUE TITLE
[BUG]: Resources using the removed legacy projects API are still present and not deprecated in the provider

### ISSUE THREAD

**Issue by stevehipwell** [integrations/terraform-provider-github#2494]
### Expected Behavior

Resources which will not work should not be in the provider and should have been deprecated when the API deprecation announcement was issued.

### Actual Behavior

The resources using the removed legacy projects API are still present and not deprecated in the provider.

### Terraform Version

n/a

### Affected Resource(s)

- `github_organization_project`
- `github_project_card`
- `github_project_column`
- `github_repository_project`

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by stevehipwell** [comment]
The code to deprecate this is currently in #2476 but I'm planning on opening a seperate PR for this change before that gets merged.

**Comment by stevehipwell** [comment]
Yes it's a breaking change due to the SDK changes, although it's been broken for a long time. The aim 5 months ago was to get a version out with a deprecation notice before changing the SDK and releasing a major version bump. Unfortunately this repo has effectively been abandoned.

**Comment by emouawad** [comment]
mm.. abandoned?! this is github provider we are talking about - do you mean development has shifted to another repo? what's the alternative?

**Comment by stevehipwell** [comment]
@emouawad check out the commit history and lack of action on issues or PRs.
